test/ios_coverage_test.sh

```diff
@@ -173,6 +183,13 @@ function test_standalone_unit_test_coverage_json() {
       grep '"name":"SharedLogic.m:-\[SharedLogic doSomething\]"'
 }
 
+function test_standalone_unit_test_coverage_coverage_manifest() {
+  create_common_files
+  do_coverage ios --test_output=errors --ios_minimum_os=9.0 --experimental_use_llvm_covmap --action_env=LCOV_MERGER=/usr/bin/true //app:coverage_manifest_test || fail "Should build"
+  assert_contains "SharedLogic.m:-\[SharedLogic doSomething\]" "test-testlogs/app/coverage_manifest_test/coverage.dat"
+  assert_contains "SF:./app/SharedLogic.m" "test-testlogs/app/coverage_manifest_test/coverage.dat"
```

Comment:
Is it worth adding a `assert_not_contains` to assure that the files outside of `coverage_manifest.txt` aren't present?

Reply:
yea good spot, i have a follow up change for this